### PR TITLE
name update

### PR DESCRIPTION
**Pull request by Messy64**
Name update/change

### CODE REVIEW

**Review by seanmlafferty — APPROVED** [review]
Looks good.

File: storage.ps1 [hunk 1]
```diff
@@ -1 +1 @@
-new-azstorageaccount
\ No newline at end of file
+new-azstorageaccount -name test
```
Comment (srijana02) [inline storage.ps1:1]: test

**Review by seanmlafferty — APPROVED** [review]
Good job.

File: storage.ps1 (hunk 1 above)
Comment (Messy64) [reply storage.ps1:1]: test two

**Review by srijana02 — APPROVED** [review]

**Pull request merged**